stac_check/lint.py

```diff
@@ -33,6 +40,18 @@ def __post_init__(self):
         self.file_name = os.path.basename(self.item).split('.')[0]
         self.best_practices_msg = self.create_best_practices_msg()
 
+    @staticmethod
+    def parse_config(config_file):
+        default_config_file = os.getenv("STAC_CHECK_CONFIG", "stac-check.config.yml")
+        with open(default_config_file) as f:
+            default_config = yaml.load(f, Loader=yaml.FullLoader)
```

Comment:
If `STAC_CHECK_CONFIG` isn't set, and `stack-check.config.yml` doesn't exist in the user's current working directory, this will error, no?

You might need to use something like `pkg_resources` to include the configuration file in the installed package (one example of using `pkg_resources` in such a way is here: https://github.com/stactools-packages/noaa-c-cap/blob/main/src/stactools/noaa_c_cap/utils.py).

Reply:
Good call! I think it's fine now with the MANIFEST.in file. Thank you